src/documents/tutorial-prefsFramework/CreatingAPrefsEditor.md

```diff
@@ -0,0 +1,779 @@
+Infusion [Renderer](../Renderer.md) to render the view based on data in the component’s model.
+The _[protoTree](../RendererComponentTrees.md)_ is the instructions for how the data in the component’s model maps to the template.
+Let’s look at this more closely:
+<pre class="highlight">
+<code class="hljs javascript">protoTree: {
+    autoPilot: "${autoPilot}"
+}</code></pre>
+
+A protoTree contains key/value pairs, where
+* the key is a selector _name_ specified in the `selectors` option, and
+* the value is the specification for what to render into the DOM node referenced by the selector.
+
+Here, the one key `autoPilot` refers to the selector named autoPilot
+i.e. the reference to the checkbox in the HTML template.
+The value is a reference to the `autoPilot` property of Panel’s data model.
+In the Infusion Framework, an _IoC reference_ (IoC stands for Inversion of Control) is a reference
+to an object in the current context using a particular syntax – specifically, the form `{context-name}.some.path.segments`.
+_Coming soon: More information about IoC references._
+</dd>
+</dl>
+
+#### Auxiliary Schema ####
+The [Auxiliary Schema](../AuxiliarySchemaForPreferencesFramework.md) is a document that specifies
+all the things needed to actually build the Preference Editor.
+The Auxiliary Schema for our example Editor
+is defined in the `schemas/auxiliary.js` file:
+
+```javascript
+fluid.defaults("minEditor.auxSchema", {
+    gradeNames: ["fluid.prefs.auxSchema"],
+    auxiliarySchema: {
+          // some code not shown
+    }
+});
+```
+Again, we use `fluid.defaults()` to create the Schema.
+As with the Primary Schema and the Panel, `fluid.defaults()` is passed two arguments:
+1) a string name (`"minEditor.auxSchema"`), and 2) a JavaScript object literal
+containing configuration options.
+
+Let’s look at the Schema itself in detail:
+```javascript
+auxiliarySchema: {
+    // the loaderGrade identifies the "base" form of preference editor desired
+    loaderGrades: ["fluid.prefs.fullNoPreview"],
+
+    // 'terms' are strings that can be re-used elsewhere in this schema;
+    terms: {
+        templatePrefix: "html"
+    },
+
+    // the main template for the preference editor itself
+    template: "%templatePrefix/minEditor.html",
+
+    autoPilot: {
+        // this 'type' must match the name of the pref in the primary schema
+        type: "minEditor.autoPilot",
+        panel: {
+            // this 'type' must match the name of the panel grade created for this pref
+            type: "minEditor.panels.autoPilot",
+
+            // selector indicating where, in the main template, to place this panel
+            container: ".mec-autoPilot",
+
+            // the template for this panel
+            template: "%templatePrefix/autoPilot.html"
+        }
+    }
+}
+```
+An auxiliary schema can be generally divided into two types of properties:
```

Comment:
"type" is used a lot in this section to refer to the component or gradename. Could we use "kinds" or some other synonym here to reduce confusion?